dropwizard-parent-pom/pom.xml

```diff
@@ -252,11 +251,6 @@
       </dependency>
       <dependency>
         <groupId>io.dropwizard</groupId>
-        <artifactId>dropwizard-spdy</artifactId>
```

Comment:
Ahh i see what you mean, I forgot we define docker in both ECS parent and this one, never mind - carry on. That said I think using the bom is an improvement.